FileShareService.DesktopClient.Core/Jobs/SetExpiryDateJob.cs

```diff
@@ -1,12 +1,21 @@
-﻿using System;
+﻿using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
 
 namespace UKHO.FileShareService.DesktopClient.Core.Jobs
 {
     public class SetExpiryDateJob : IJob
     {
+        public const string JobAction = "setExpiryDate";
         public string DisplayName { get; set; }
 
         public SetExpiryDateJobParams ActionParams { get; set; } = new SetExpiryDateJobParams();
+        public List<string> ErrorMessages { get ; set ; }
+
+        public List<string> Validate(JToken jsonToken)
+        {
+            throw new NotImplementedException();
```

Comment:
`Validate()` will be called by `JobsParser` so must not throw an exception. Please remove the throw

Reply:
Removed throw and kept it as blank. The validations will be implemented when the corresponding job PBI will be picked up.